test/helpers.js

```diff
@@ -591,3 +592,19 @@ export const itReturnAllResultsWhenAnUnknownFilterIsUsed = ({
     });
   });
 };
+
+export const expectNewNotificationToBeAdded = (userId, notification) => {
+  let userLatestNotification;
+
+  beforeEach(async () => {
+    userLatestNotification = await Notification.find({ userId }).sort({ _id: -1 }).limit(1);
+  });
+
+  context('when new notification is added', () => {
+    it('adds a new notification for user', async () => {
```

Comment:
This doesn't seem like an async function

```suggestion
    it('adds a new notification for user', () => {
```